docs/gds-supported-platform/getting-started-gsp-local.md

```diff
@@ -71,34 +123,46 @@ This process is resource-intensive and you must set the resource amount used by
     docker build . --tag prototype-kit:latest
     ```
 
-1. Test the Docker image:
+1. Check that the Dockerised prototype kit runs successfully:
 
     ```
-    docker run --publish 3000:3000 prototype-kit:latest
+    docker run -it --rm --publish 3000:3000 prototype-kit:latest
     ```
-    This will create a local Docker container with the app running in that container.
+    This will create a local Docker container with the prototype kit at http://localhost:3000/.
 
+    Enter `Ctrl-c` to stop the container once you have confirmed that the prototype kit is running.
 
-1. Build the Docker image in the GSP cluster (gsp-local):
+1. Build the Docker image in the GSP instance:
 
-    Talk to cluster 'docker' (not the version you have locally installed)
     ```
     eval $(minikube docker-env -p gsp-local)
-    docker ps (should yield screens full of containers)
     docker build . -t prototype-kit:latest
     ```
-    You have created a Docker image and copied it into your local GSP cluster.
-    
+
+    You have created a Docker image and built it into your local GSP instance.
 
 ## Create a Helm chart
 
-The GDS Supported Platform uses a packaging format called [Helm charts](https://helm.sh/docs/developing_charts/). A chart is a collection of files that describe a related set of Kubernetes resources.
+Kubernetes resources describe the configuration of your running app. You define these resources as `.yaml` files and collect them together in a packaging format called a [Helm chart](https://helm.sh/docs/developing_charts/).
 
 You create Helm charts as files in a directory. These files are then packaged into versioned archives that users can deploy.
```

Comment:
We are not actually using the archives feature here in this example so this might be confusing, suggest:

"You create Helm charts as files in a directory."

Reply:
agree